stylesheets/_session.scss

```diff
@@ -342,4 +342,8 @@ input {
 
 .module-message__text {
   white-space: pre-wrap;
+
+  svg {
+    margin-right: var(--margins-xs);
```

Comment:
Should this not be margin-end? Or is it always on the right?

Reply:
fixed